src/app/flows/fines/fines-mac/services/fines-mac-payload/fines-mac-payload.service.ts

```diff
@@ -30,6 +30,7 @@ import { ITransformItem } from '@hmcts/opal-frontend-common/services/transformat
 import { ISessionUserState } from '@hmcts/opal-frontend-common/services/session-service/interfaces';
 import { IOpalFinesDraftAccountPatchPayload } from '@services/fines/opal-fines-service/interfaces/opal-fines-draft-account.interface';
 import { OPAL_FINES_DRAFT_ACCOUNT_STATUSES } from '@services/fines/opal-fines-service/constants/opal-fines-draft-account-statues.constant';
+import { FINES_MAC_DEFENDANT_TYPES_KEYS } from '../../constants/fines-mac-defendant-types-keys';
 
 @Injectable({
   providedIn: 'root',
```

Comment:
Should we change this to `any`

```suggestion
  providedIn: 'any',
```